views/blog/post/__form.php

```diff
@@ -41,5 +48,61 @@
                   placeholder="Content" required><?= Html::encode($body['content'] ?? '') ?></textarea>
     </div>
 
+    <div class="form-group">
+        <label for="AddTag">Add tag</label>
+        <input type="text" class="form-control" id="AddTag" placeholder="Add tag" value="">
+        <?= Html::button(
+    'Add',
+    ['class' => 'btn btn-primary mt-2', 'onclick' => <<<JS
```

Comment:
It is better to have it in a separate JS-file. Likely, asset bundles is a good match to register it: https://github.com/yiisoft/app/tree/master/src/Asset